README.md

```diff
@@ -84,6 +84,24 @@ The color-scheme mixin builds all relevant color stylings you need to create a f
 
 For example take a look at the calendar icon on the index page.
 
+## Contributing
+We are very happy about you contributing and have defined a process for that, which makes it easy for us to maintain the website. 
```

Comment:
We are very happy about you contributing to the Hoodie website! We have defined a process that makes it easy for you to contribute and helps us maintain the website.